test.js

```diff
@@ -55,3 +55,36 @@ test('textarea values get copied', function (t) {
   yo.update(el, textarea('bar'))
   t.equal(el.value, 'bar')
 })
+
+test('user supplied onBeforeElUpdated is called', function (t) {
+  t.plan(1)
+  function yolo (e) {
+    t.fail('Event handler should not be called')
+  }
+  var el = yo`<div>Hey there,</div>`
+  var newEl = yo`<div onclick=${yolo}>How you doin'?</div>`
+  var opts = {
+    events: false,
+    onBeforeElUpdated: function(from, to) {
+      t.ok('User supplied onBeforeElUpdated called')
+    }
+  }
+  yo.update(el, newEl, opts)
+  el.click()
+})
+
+test('user supplied onBeforeElUpdated can skip copying of events', function (t) {
+  t.plan(2)
+  function a () { t.ok(true, 'called a') }
+  function b () { t.fail('this event handler should not be called') }
+  var el = yo`<button onclick=${a}>hi</button>`
+  var opts = {
+    events: false,
+    onBeforeElUpdated: function(from, to) {
+      t.ok('User supplied onBeforeElUpdated called')
+      return false
```

Comment:
Returning `false` from user supplied `onBeforeElUpdated` will skip `yo-yo`'s copying of events.